backend/danswer/key_value_store/store.py

```diff
@@ -28,7 +28,8 @@
 
 class PgRedisKVStore(KeyValueStore):
     def __init__(self) -> None:
-        self.redis_client = get_redis_client()
+        tenant_id = current_tenant_id.get()
```

Comment:
Is there any way to inject these global dependencies in instead of creating a hard dependency on them?